cg/models/cg_config.py

```diff
@@ -162,6 +162,8 @@ class RnafusionConfig(CommonAppConfig):
     profile: str
     conda_binary: Optional[str] = None
     launch_directory: str
+    tw_binary_path: str
```

Comment:
Consider not shortening the name to tw, always better to be explicit.

Reply:
changed